plugins/modules/mso_schema_site_contract_service_graph.py

```diff
@@ -0,0 +1,344 @@
+        type: list
+        elements: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+seealso:
+- module: cisco.mso.mso_schema_template_contract_service_graph
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+
+EXAMPLES = r"""
+- name: Associate a service graph with a site contract
+  cisco.mso.mso_schema_site_contract_service_graph:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    tenant: sab_ansible_tenant
+    schema: sab_ansible_schema
+    template: sab_ansible_template1
+    site: ansible_test
+    contract: Contract1
+    service_graph_schema: sab_ansible_schema
+    service_graph_template: sab_ansible_template1
+    service_graph: sab_sg
+    node_relationship:
+      - cluster_interface_device: sab_ansible_tenant_firewall1
+        provider_connector_cluster_interface: clu_if1
+        provider_connector_redirect_policy: redirect_policy1
+        consumer_connector_cluster_interface: clu_if1
+        consumer_connector_redirect_policy: redirect_policy1
+      - cluster_interface_device: sab_ansible_tenant_adc
+        provider_connector_cluster_interface: clu_if3
+        provider_connector_redirect_policy: redirect_policy1
+        consumer_connector_cluster_interface: clu_if3
+        consumer_connector_redirect_policy: redirect_policy1
+        consumer_subnet_ips: ["1.1.1.1/24", "4.4.4.4/24"]
+      - cluster_interface_device: sab_ansible_tenant_other
+        provider_connector_cluster_interface: clu_if4
+        provider_connector_redirect_policy: redirect_policy1
+        consumer_connector_cluster_interface: clu_if4
+        consumer_connector_redirect_policy: redirect_policy1
+    state: present
+
+- name: Query a site contract service graph with contract name
+  cisco.mso.mso_schema_site_contract_service_graph:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: sab_ansible_schema
+    template: sab_ansible_template1
+    contract: Contract1
+    site: ansible_test
+    state: query
+  register: query_result
+
+- name: Query all site contract service graphs associated with a site template
+  cisco.mso.mso_schema_site_contract_service_graph:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: sab_ansible_schema
+    template: sab_ansible_template1
+    site: ansible_test
+    state: query
+  register: query_result
+
+- name: Remove a site contract service graph with contract name
+  cisco.mso.mso_schema_site_contract_service_graph:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: sab_ansible_schema
+    template: sab_ansible_template1
+    site: ansible_test
+    contract: Contract1
+    state: absent
+"""
+
+RETURN = r"""
```

Comment:
Do we provide return documentation for ansible-mso?

Reply:
Most of the MSO modules do not have return value. So I keeping it empty for now.